.github/workflows/integration_tests.yml

```diff
@@ -95,7 +96,7 @@ jobs:
       - name: Make Steward artifact available to other jobs
         uses: actions/upload-artifact@v3
         with:
-          name: ${{ env.IMAGE_NAME }}
+          name: ${{ env.IMAGE_NAME }}-pr-${{ env.PR_NUMBER }}
```

Comment:
I feel like we talked about this before, but do we need to include the PR number in the path as well to ensure it's unique?

Reply:
my bad